### ISSUE TITLE
only jamming

### ISSUE THREAD

**Issue by Parker84757** [wfhstudio/RollJam-Arduino#2]
I am set up with your schematic and have checked it three times over and verified all the solder joints are good, I am not able to ever receive any signals from my keyfobs, Ive tried multiple fobs.... Do you know why this may be? I have changed the modulation in the ino file and i am verifying my work with a hackrf and flipper zero, they are able to see the fobs signal, just not the Arduino device. Ever since watching sammys youtube video  I have been obsessed with this rolljam idea and have spent hundreds on components and am determined to make it work, any guidance would be the most appreciated.

**Comment by Dobermannisland** [comment]
hi my friedn have a pen sdr for check signal ??  have firmware work ?

Il giorno mer 12 giu 2024 alle ore 10:00 Parker84757 <
***@***.***> ha scritto:

**Comment by Parker84757** [comment]
Yes I have been using a portapack hackrf to watch for the signal as well as
a flipper zero to transmit known signals I've seen no changes to the air
and the signals that are transmitting. I have a little usb device that when
plugged into a powerbank jams the shit out of the whole 2.4 wavelength and
when i place it inbeteen the bluetooth device and source it will cut out
the transmission but it only has a pcb printed antenna. i bought it from
liexpress to plug into my work van bc my business partner has been pissing
me off getting new gps for the work vehicles of our employees and insisted
we install one on my van as well. So to save face and look like a team
player I allowed it, but secretly am jamming everytime i drive around lol.
So I dont know why the device youve built isnt working bc i followed the
build instructions to a T and all the hardware is of quality. Any idea
whats going on?

Sorry I typed this up right after i recieved your email but forgot to send
it until now...

**Comment by Parker84757** [comment]
And yes i used the web installer for the esp32 and installed the dual nrf24
device build bin file

**Comment by Dobermannisland** [comment]
I bought this device online and it works very well


Il giorno ven 14 giu 2024 alle ore 06:01 Parker84757 <
***@***.***> ha scritto:

**Comment by Dobermannisland** [comment]
https://www.youtube.com/watch?v=XOfNEa-M1Tk

Il giorno ven 14 giu 2024 alle ore 11:38 gianni Iky ***@***.***>
ha scritto:
